src/components/AddItems.jsx

```diff
@@ -16,14 +16,33 @@ const radioInputOptions = {
 	notSoon: ['notsoon', nextPurchaseDate.notSoon, 'Not Soon'],
 };
 
-export function AddItems() {
+export function AddItems({ data }) {
 	const [listPath] = useStateWithStorage('tcl-shopping-list-path', null);
+	console.log(data);
 
 	const handleSubmit = useCallback(
 		async (event) => {
 			event.preventDefault();
 
 			const itemName = event.target.elements['item-name'].value;
```

Comment:
Looks great!
I want to propose to abstract the normalization of the item name into a function that is outside of the component and looks something like this:

export const normalizeItem = (itemName) => {
 return itemName
 .trim()
 .toLowerCase()
 .replace(/[&/\#, +$!,~%.'":*?<>{}]/g, '');
}

so that we can simple call the function and keep code dry:
const normalizedItemName = normalizeItem(itemName);